expo/gbe/forms.py

```diff
@@ -798,3 +798,138 @@ class ContactForm(forms.Form):
     email = forms.EmailField(required=True)
     subject = forms.CharField(required=True)
     message = forms.CharField(widget=forms.Textarea)
+
+
+class CostumeBidDraftForm(forms.ModelForm):
+    required_css_class = 'required'
+    error_css_class = 'error'
+
+    active_use = \
+        forms.TypedChoiceField(widget=forms.RadioSelect,
+                               choices=boolean_options,
+                               label=costume_proposal_labels['active_use'],
+                               required=False)
+    debut_date = \
+        forms.CharField(
+            label=costume_proposal_labels['debut_date'],
+            help_text=costume_proposal_help_texts['debut_date'],
+            widget=forms.TextInput(attrs={'placeholder': 'MM/YYYY'}),
+            required=False)
+
+    class Meta:
+
+        model = Costume
+        fields = ['performer',
+                  'title',
+                  'act_title',
+                  'debut_date',
+                  'active_use']
+        help_texts = costume_proposal_help_texts
+        labels = costume_proposal_labels
+
+
+class CostumeBidSubmitForm(forms.ModelForm):
+    required_css_class = 'required'
+    error_css_class = 'error'
+
+    active_use = \
```

Comment:
Please can we avoid backslash continuations? Break the line after the open parenthesis and you should be fine for pep8.